pallets/pools/src/mock.rs

```diff
@@ -302,6 +302,17 @@ impl Config for Test {
 	type MaxTranches = MaxTranches;
 	type WeightInfo = ();
 	type TrancheWeight = TrancheWeight;
+	type PoolCurrency = PoolCurrency;
+}
+
+pub struct PoolCurrency;
+impl Contains<CurrencyId> for PoolCurrency {
+	fn contains(id: &CurrencyId) -> bool {
+		match id {
+			CurrencyId::Tranche(_, _) | CurrencyId::Native => false,
```

Comment:
Are we sure we won't want to allow Native, especially on altair where we might be financing crypto-native assets?

Reply:
Whoops, Jeroen made the same comment below. All set :)